### ISSUE TITLE
Application breaks when downloading pool 19615

### ISSUE THREAD

**Issue by RowanSkie** [Vilagamer999/e621-pool-downloader.net#2]
Ha! I broke the thing.

For context, I'm downloading all of my pools, this pool has "s in the title which I expect break things.

**Comment by Vilagamer999** [comment]
yeah for some reason it didn't sanitise the text, I coded this as a sort of fun side project but I'm glad someone is using it, I'll see what I can do :)
